### ISSUE TITLE
Cannot install via Gradle

### ISSUE THREAD

**Issue by idulnyavka** [wooplr/Spotlight#53]
```
buildscript {
    repositories {
        //...
        maven { url "https://jitpack.io" }
    }

    //...
}

//...
dependencies {
    //...
    compile 'com.github.wooplr:Spotlight:1.2.3'
}
```

=> Error on Sync:

> 
> Error:Could not GET 'http://oss.jfrog.org/artifactory/oss-snapshot-local/com/github/wooplr/Spotlight/1.2.3/Spotlight-1.2.3.pom'. Received status code 409 from server: Conflict
> <a href="toggle.offline.mode">Enable Gradle 'offline mode' and sync project</a>
> 

Also, trying to open the URL above into a browser gives more details  on the error:
```
{
  "errors" : [ {
    "status" : 409,
    "message" : "The repository 'oss-snapshot-local' rejected the resolution of an artifact 'oss-snapshot-local:com/github/wooplr/Spotlight/1.2.3/Spotlight-1.2.3.pom' due to conflict in the snapshot release handling policy."
  } ]
}
```

**Comment by idulnyavka** [comment]
Same error trying to include  v 1.2.2
Android Studio 2.2.3 (Windows 10)

**Comment by idulnyavka** [comment]
Hmm... worked after I added repository to root gradle.build file under "allprojects ".
Why is it not working under project-specific gradle file?

**Comment by dhavalwooplr** [comment]
I'm using it in the project specific gradle file without any sync errors. Your error seems about some conflict.

**Comment by idulnyavka** [comment]
Nope... As I told above - it dod not work when I put repository to the project build file, but worked within app level file.
Putting back to project file raised error again.
Anyway, it compiles according to the README file, so let's consider this not an issue but a feature request.

**Comment by zacharee** [comment]
Try removing the `buildscript {}` part. I've noticed that Gradle seems to ignore repositories under this category, and instead only sees repositories if they're only under `repositories {}`.
